src/content/docs/logs/logs-everywhere/telemetry-in-context.mdx

```diff
@@ -0,0 +1,219 @@
+
+  ![APM summary errors](./images/demotron1-logs030722.png "APM summary errors")
+
+  **How can they quickly decide what's related to the problem?**
+  
+  They have not set up log collection. But the Ruby agent for their app has been recently updated to include telemetry in context, so the app now automatically receives metrics about its logs. The metrics are faceting the count by a severity field commonly included in their Rails framework.
+
+  ![Logs summary](./images/demotron2-logs030722.png "Logs summary")
+
+  **What entities are related?**
+  
+  On the same UI page, they click the **Logs** chart, so they can review the **Log Summary** view for entities in the New Relic One Explorer. This view shows them when instrumented logs have been collected and when those logs had an `Error` severity level or worse. It also shows them a list of log patterns and what percentage of all logs fit each pattern.
+
+    </Collapser>
+
+    <Collapser
+    id="patterns"
+    title="What patterns emerge in the logs?"
+    >
+
+    In the **Log Patterns** UI, rare error messages have begun to occur frequently, and they now appear in the list. This helps the engineer to focus on problematic logs instead of all the noisy status updates.
+
+    ![Log patterns](./images/demotron3-logs030722.png "Log patterns")
+
+    **What patterns should they focus on?**
+    
+    The engineer notices an interesting log pattern and wants to view only the logs that fit this pattern. By clicking on it, this adds the pattern's value as a filter to the logs being displayed. This narrows the focus.
+
+    </Collapser>
+
+    <Collapser
+    id="log-details"
+    title="What can more log details reveal?"
+    >
+
+    The engineer wants to see all the values contained in the log record, so they click the **Log Detail** view. This helps them validate that the log itself is meaningful. It also lets them drill down further into either the Kubernetes environment where the app runs, or directly to any distributed traces or APM errors related to the log record.
+
+    ![Log details](./images/demotron4-logs030722.png "Log details")
+    
+    The engineer decides to look at the APM error where they can see a full stack trace. They can also see logs from their **Errors in-box** UI.
+    
+    ![Error logs](./images/demotron5-logs030722.png "Error logs")
+
+    </Collapser>
+
+    <Collapser
+    id="log-sampling"
+    title="What additional test data will help troubleshoot?"
+    >
+
+    The troubleshooting team has been running tests to isolate the cause, but not all logs have been collected for the test transactions. The `AlwaysOn` default log collection has been useful to validate that the services have an issue and to examine it initially. But now they need to turn up the sampling to continue troubleshooting.
```

Comment:
What does `AlwaysOn` refer to? 

If you'd like to add the process to turn up sampling: 

To adjust their sampling, they'll update the configuration `application_logging.forwarding.max_samples_stored` to a higher number. It defaults to 10,000 logs stored per minute.

Reply:
When agent teams were talking about the default being always on, and I think I interpreted that as a config value called AlwaysOn. I'll double-check my notes, and in the meantime I will add your recommendation. Thanks for making this accurate!